test/integration/integration_admin_test.cc

```diff
@@ -479,27 +476,35 @@ class StatsMatcherIntegrationTest : public HttpIntegrationTest, public testing::
 TEST_F(StatsMatcherIntegrationTest, ExcludePrefixServerDot) {
   stats_matcher_.mutable_exclusion_list()->add_patterns()->set_prefix("server.");
   initialize();
+  makeRequest();
```

Comment:
any particular reason not to fold makeRequest() and verifyResponse() into the same function? It looks like it'd be fine to do the HasSubstr check after the body of verifyResponse().

Reply:
Thanks, I will do this.